### PR TITLE
Fix comments problem.

### PR DESCRIPTION
**Pull request by hisa0507**
Sorry for my commits. I changed these comments if they're not implemented in CuPy compared with NumPy. If you think about anything in writing code or comments of mine, please let me know about that. Thank you.

### CODE REVIEW

**Comment by hisa0507** [comment]
Oops, something went wrong.. Building up other branch..

**Comment by hisa0507** [comment]
see #1998

**Comment by kidsHero** [comment]
Please read the #1018 comment carefully.
You stole a excellent software developer's time and the author of the original software.
Give me a break!

**Comment by delta2323** [comment]
@kidsHero Hi. This is also developer team's problem caused by our insufficient checks. We will pay more attentions to such copyright matters. Thank you for taking time to point out the problem.

**Comment by delta2323** [comment]
Could you add a comment in somewhere in `cupy/indexing/generate.py` that the implementation is referenced from NumPy's one?

File: cupy/indexing/generate.py [hunk 1]
```diff
@@ -1,13 +1,162 @@
...
+    def __len__(self):
+        return 0
+
+
+class CClass(AxisConcatenator):
+
+    def __init__(self):
+        """Translates slice objects to concatenation along the second axis.
+
+        This is CuPy object that corresponds to 'cupy.r_', which is
```
Comment (delta2323) [inline cupy/indexing/generate.py]: This is *a* CuPy object ...

Comment (kidsHero) [inline cupy/indexing/generate.py]: LMAO

**Comment by delta2323** [comment]
@beam2d @okuta Should we merge after #2035 is ready, or can I merge this before?

**Comment by delta2323** [comment]
Now #2035 is merged, I merge this PR to reverted branch.

**Pull request merged**